cookbook/integrating-custom-filters.rst

```diff
@@ -0,0 +1,117 @@
+Integrating custom filters
+==========================
+
+When you want to enable custom filters for your bundle you have to follow the 
+following steps. It's important to know that the filter component work with 
+contexts. This means for example that the lists of contacts has the 
+```contacts``` context and everything concering the filters for the list will 
+need this context. It should therefore be unique.
+
+1. Add the missing data types to the field descriptors
```

Comment:
I would use captions here (underline with `-`) instead of a numbered list. Same for the other steps.